src/module/apps/acc_diff/plugins/combinedArms.ts

```diff
@@ -0,0 +1,147 @@
+import { AccDiffHudData, AccDiffHudTarget } from "../data";
+import { AccDiffHudCheckboxPluginData, AccDiffHudPluginCodec } from "./plugin";
+import { enclass } from "../../serde";
+import { AccDiffWindowType, WeaponType } from "../../../enums";
+import { slugify } from "../../../util/lid";
+import { AbstractTalent } from "./abstractTalents";
+import { LancerItem } from "../../../item/lancer-item";
+import { LancerActor } from "../../../actor/lancer-actor";
+import { getHistory } from "../../../util/misc";
+
+//Automated
+//A lot of common talent boilerplate is contained in SampleTalent
+export class CombinedArms_2 extends AbstractTalent implements AccDiffHudCheckboxPluginData {
+  //AccDiffHudPlugin requirements
+  static slug: string = slugify("CQB-Trained", "-");
+  slug: string = slugify("CQB-Trained", "-");
+  static kind: "hase" | "attack" = "attack";
+  kind: "hase" | "attack" = "attack";
+  lid: string = "t_combined_arms";
+  talentRank: number = 2;
+  humanLabel: string = "CQB-Trained";
+  quickReference: string = "*";
+  tooltip: string = "You don’t gain Difficulty from being Engaged.";
+
+  //AccDiffHudPlugin requirements
+  // the codec lets us know how to persist whatever data you need for rerolls
+  static get codec(): AccDiffHudPluginCodec<CombinedArms_2, unknown, unknown> {
+    return enclass(this.schemaCodec, CombinedArms_2);
+  }
+
+  //We don't care about targets
+  static perRoll(item?: LancerItem | LancerActor): CombinedArms_2 {
+    let ret = new CombinedArms_2();
+    return ret;
+  }
+
+  //The unique logic of the talent
+  //Name defined from SampleTalent
+  talent(data: AccDiffHudData, target?: AccDiffHudTarget) {
+    if (!data.weapon.engaged && !data.lancerActor?.system.statuses.engaged) return;
+    if (data.weapon.weaponType === WeaponType.Melee) return;
+
+    this.active = true;
+  }
+
+  get visible(): boolean {
+    //This talent does not apply to tech attacks
+    if (this.data?.windowType === AccDiffWindowType.Tech) return false;
+
+    return true;
+  }
+
+  //RollModifier Requirements
+  //Modify accuracy
+  get accBonus(): number {
+    //How to mimic not gaining difficulty from Engaged?
+    //Just add 1 Accuracy, duh
+    if (this.active) {
+      return this.data?.weapon.engaged ? 1 : 0;
+    }
```

Comment:
Yeah, that's probably the best. The alternative would have to be in the Engaged automation, checking for the talent there and then disabling the checkbox... In other words, completely different from how the rest of the talents are automated here.